rspace/src/main/scala/coop/rchain/rspace/InMemoryOps.scala

```diff
@@ -10,7 +10,7 @@ trait InMemTransaction[S] {
   def name: String
 }
 
-trait InMemoryOps[S] {
+trait InMemoryOps[S] extends CloseOps[InMemTransaction[S]] {
```

Comment:
Shouldn't the inheritance be done in the other direction? Then you could call `super` in CloseOps and the confusing `Imp` suffix in methods would be unnecessary

Reply:
Makes sense. My original intent was to guarantte order of calls for all possible IStore implementations. The best approach would be based on something like "trait CloseOps[T <: TransactionOpsBase] extends T", but scala prohibits such kind of inheritance, so I decided to finalize base methods and use *Imp functions for implementations.
However, from one side the *Imp based approach guarantees that there is no way to forget to call .super methods first, but from other side it's an overkill for just 2 one-line implementations (LMDB and InMemory) completely covered by unit-tests. Looks like code is really simplified without it.